gtkm/auth/service_auth.py

```diff
@@ -1,13 +1,67 @@
-from fastapi import APIRouter
+from fastapi import APIRouter, Depends
+from fastapi.encoders import jsonable_encoder
+from fastapi.responses import RedirectResponse, JSONResponse
+from sqlalchemy.orm import Session
+import requests
+import uuid
 
+import crud, model, schemas
```

Comment:
Popraw ten import.  
wydaje mi się, że tu powinny być względne importy aby działało tzn 
`.crud, .model, .schemes`

Reply:
Poza tym rozbiłbym to na kilka linijek

Reply:
Importy zamienione na konkretne funkcje i obiekty + rozdzielone na osobne linijki